contracts/src/libraries/RewardMerkleTreeVerifier.sol

```diff
@@ -17,10 +19,6 @@ library RewardMerkleTreeVerifier {
 
     uint256 public constant TREE_DEPTH = 160;
 
-    struct AccruedRewardsProof {
-        bytes32[] siblings;
-    }
-
     function _hashLeaf(uint256 value) internal pure returns (bytes32) {
         bytes32 firstHash = keccak256(abi.encodePacked(value));
```

Comment:
we're good on using encodePacked here?

Reply:
Since it is only called once (as opposed to 160 times for the internal nodes) I didn't bother with assembly, happy to change it though.